packages/react/src/components/primitive/clear-button.tsx

```
@@ -0,0 +1,48 @@
+  return (
+    <Comp
+      {...props}
+      data-state={state}
+      aria-disabled={disabled}
+      onClick={(e) => {
+        onClick?.(e);
+        setFiles([]);
+      }}
+      ref={ref}
+    >
+      <PrimitiveSlot default="Clear" children={children} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)